src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingController.kt

```diff
@@ -1,81 +1,52 @@
 package no.nav.familie.ef.sak.ekstern
 
-import no.nav.familie.ef.sak.behandling.BehandlingRepository
-import no.nav.familie.ef.sak.behandling.domain.BehandlingType
 import no.nav.familie.ef.sak.fagsak.domain.Stønadstype
 import no.nav.familie.ef.sak.opplysninger.personopplysninger.PdlClient
 import no.nav.familie.ef.sak.opplysninger.personopplysninger.pdl.identer
 import no.nav.familie.kontrakter.felles.PersonIdent
 import no.nav.familie.kontrakter.felles.Ressurs
 import no.nav.security.token.support.core.api.ProtectedWithClaims
 import org.springframework.http.MediaType
-import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
-@RequestMapping(path = ["/api/ekstern/behandling"],
-                consumes = [MediaType.APPLICATION_JSON_VALUE],
-                produces = [MediaType.APPLICATION_JSON_VALUE])
-@Validated
+@RequestMapping(
+        path = ["/api/ekstern/behandling"],
+        consumes = [MediaType.APPLICATION_JSON_VALUE],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+)
 class EksternBehandlingController(private val pdlClient: PdlClient,
-                                  private val behandlingRepository: BehandlingRepository) {
+                                  private val eksternBehandlingService: EksternBehandlingService) {
 
-
-    /**
-     * Blir brukt av mottak for å sjekke om en perosn allerede har en behandling i ef-sak
-     * Kunde ha flyttet ut funksjonaliteten i en egen service,
-     * men for å unngå att andre bruker den (med kall mot pdl) så ble alt her
-     *
-     * Hvis man ikke sender type blir alle typer sjekket om det finnes noen.
-     */
     @PostMapping("finnes")
     @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
     fun finnesBehandlingForPerson(@RequestParam("type") stønadstype: Stønadstype?,
                                   @RequestBody request: PersonIdent): Ressurs<Boolean> {
         val personidenter = pdlClient.hentPersonidenter(request.ident, historikk = true).identer()
-        return finnesBehandlingFor(personidenter, stønadstype)
+        if (personidenter.isEmpty()) {
+            return Ressurs.failure("Kunne ikke hente personidenter fra pdl")
+        }
+        return Ressurs.success(eksternBehandlingService.finnesBehandlingFor(personidenter, stønadstype))
     }
 
     /**
      * Hvis man har alle identer til en person så kan man sende inn alle direkte, for å unngå oppslag mot pdl
      * Dette er alltså ikke ett bolk-oppslag for flere ulike personer
      */
-    @PostMapping("finnes/flere-identer")
+    @PostMapping("harstonad/flere-identer")
```

Comment:
For å unngå breaking change trenger vi fortsatt å beholde den tidligere url'en. Ettersom ingen annen bruker den burde vi kanskje ha 2 url'er i mappingen? Og sen endre i ef-personhendelse? 
Burde URL gi en bilde av att det gjelder siste året eller noe slikt?

Reply:
To url-er eller en ny funksjon som faktisk gjør det vi ønsker å oppnå, men beholde den gamle til vi har skrudd over (om det er behov for dét?)

Reply:
Men er det ikke bare personhendelser som bruker denne ? (Og som skal bruke den nye funksjonen som gjør det vi vil oppnå ?)

Reply:
Legger til to stk frem til vi får "switchet" i personhendelser